src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -264,19 +264,11 @@ protected override void ProcessRecord()
 
                 if (_jschema != null)
                 {
-                    EvaluationResults evaluationResults = _jschema.Evaluate(parsedJson, new EvaluationOptions { OutputFormat = OutputFormat.List });
+                    EvaluationResults evaluationResults = _jschema.Evaluate(parsedJson, new EvaluationOptions { OutputFormat = OutputFormat.Hierarchical });
                     result = evaluationResults.IsValid;
                     if (!result)
                     {
                         HandleValidationErrors(evaluationResults);
```

Comment:
Please remove the `if` since the check is in HandleValidationErrors() now.

Reply:
Doesn't it save us a jump to the method?

Reply:
If you say about a performance now we call the small method once so the method will be inlined.

Reply:
I also think it looks weird to call "HandleValidationErrors" when we just learned there aren't any errors.

Reply:
No weird, this is often used in Runtime. We can even remove this method and inline the code.